CMakeLists.txt

```diff
@@ -64,21 +64,21 @@ build_library(sycl_blas SHARED)
 target_include_directories(sycl_blas PUBLIC ${SYCLBLAS_INCLUDE} ${THIRD_PARTIES_INCLUDE})
 set_target_properties(sycl_blas PROPERTIES VERSION ${PROJECT_VERSION})
 
-install(TARGETS sycl_blas 
-LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/sycl_blas/lib
```

Comment:
As explained offline, it is not needed because CMake will install there by default, and for some reason it was breaking the build on Jenkins (you can ask Nicolas more details about this).